modules/gapi/samples/api_ref_snippets.cpp

```diff
@@ -47,10 +47,8 @@ int main(int argc, char *argv[])
     //! [graph_decl_apply]
 
     //! [apply_with_param]
-    cv::gapi::GKernelPackage kernels = cv::gapi::combine
-        (cv::gapi::core::fluid::kernels(),
-         cv::gapi::imgproc::fluid::kernels(),
-         cv::unite_policy::KEEP);
+    cv::gapi::GKernelPackage kernels = cv::gapi::combine(cv::gapi::core::fluid::kernels(),
+                                                         cv::gapi::imgproc::fluid::kernels());
```

Comment:
Did you check how this method looks like in the documentation now?

Reply:
Fix documentation

Reply:
I mean, please keep the source alignment. It is supposed to fit the screen when viewed in the documentation.